### ISSUE TITLE
When generating CRDs from Helm, linebreak before apiVersion: apiextensions.k8s.io/v1 is omitted 

### ISSUE THREAD

**Issue by havard-fjaer** [solo-io/gloo-portal-issues#135]
**Describe the bug**
The first line `apiextensions.k8s.io/v1` for CRD `portals.portal.gloo.solo.io` becomes part of a comment on the line above.

**To Reproduce**
When running: `helm template gloo-portal gloo-portal/gloo-portal -n gloo-portal --values helm-values.yaml --version v1.1.2
`

with helm values:

```
glooEdge:
  enabled: true
crd:
  create: true
licenseKey:
...
```

`apiVersion: apiextensions.k8s.io/v1` becomes part of the above comment, and installation of the CRD fails in the cluster:

```
---
# Source: gloo-portal/templates/crd.yaml
# Generate CRDs inside the templates directory to support Helm 2 installsapiVersion: apiextensions.k8s.io/v1
kind: CustomResourceDefinition
metadata:
  labels:
    app: gloo-portal
    app.kubernetes.io/name: gloo-portal
  name: portals.portal.gloo.solo.io
...
```

**Expected behavior**
`apiVersion: apiextensions.k8s.io/v1` should be on a line by itself.

**Mitigation**
When running: `helm template gloo-portal gloo-portal/gloo-portal -n gloo-portal --values helm-values.yaml --include-crds --version v1.1.2` 
with crd/create:false: 

```
glooEdge:
  enabled: true
crd:
  create: false
licenseKey:
...
```

the CRD is generated as expected. The comment: `# Generate CRDs inside the templates directory to support Helm 2 installs` is not part of the generated templates when only using the `--include-crds` flag.

None of the other CRDs are affected by this problem. Suspect there is a line break missing in or somewhere near the template.

**Additional context**
Add any other context about the problem here, e.g.
- Gloo Portal v1.1.2
- Kubernetes version: N/A
- Helm: version.BuildInfo{Version:"v3.7.0", GitCommit:"eeac83883cb4014fe60267ec6373570374ce770b", GitTreeState:"clean", GoVersion:"go1.16.8"}
- Gateway technology and version: Gloo Edge Enterprise v1.9.3

**Comment by chrisgaun** [comment]
Probably a good first issue. We have some new engineers starting so hopefully they can pick it up

**Comment by bewebi** [comment]
Resolved by solo-io/dev-portal#2198, to be released in the next 1.3.0 beta
